robot/demos/SkidSteerServos/SkidSteeringDC.ino

```diff
@@ -0,0 +1,93 @@
+#include <ArduinoBlue.h>
+#include <SoftwareSerial.h>
+#include <Servo.h>
+
+int prevThrottle = 49;
+int prevSteering = 49;
+int throttle, steering;
+// Initialize Left & Right Velocity variables to Rest
+int VR = 127;
```

Comment:
You'll see in my code that I actually make a macro definition for the stopping speed because I've seen that in some cases (mostly with continuous servos that you can tune manually) it's not necessarily a value of 127 that stops the motor. Makes it nicer to port the code to new things if you can change the value at the top and it fixes it everywhere in your code.